src/main/webapp/WEB-INF/tags/instructor/course/courseInformation.tag

```diff
@@ -0,0 +1,139 @@
+
+<div class="well well-plain" id="courseInformationHeader">
+    <button type="button" data-original-title="Give a comment about all students in the course"
+            <c:forEach items="${giveCommentButton.attributes}" var="attribute" >
+                ${attribute.key}="${attribute.value}"
+            </c:forEach> >
+        ${giveCommentButton.content}
+    </button>
+            
+    <div class="form form-horizontal">
+        <div class="form-group">
+            <label class="col-sm-3 control-label">Course ID:</label>
+            <div class="col-sm-6" id="courseid">
+                <p class="form-control-static">${courseDetails.course.id}</p>
+            </div>
+        </div>
+        
+        <div class="form-group">
+            <label class="col-sm-3 control-label">Course name:</label>
+            <div class="col-sm-6" id="coursename">
+                <p class="form-control-static">${courseDetails.course.name}</p>
+            </div>
+        </div>
+        
+        <div class="form-group">
+            <label class="col-sm-3 control-label">Sections:</label>
+            <div class="col-sm-6" id="total_sections">
+                <p class="form-control-static">${courseDetails.stats.sectionsTotal}</p>
+            </div>
+        </div>
+        
+        <div class="form-group">
+            <label class="col-sm-3 control-label">Teams:</label>
+            <div class="col-sm-6" id="total_teams">
+                <p class="form-control-static">${courseDetails.stats.teamsTotal}</p>
+            </div>
+        </div>
+        
+        <div class="form-group">
+            <label class="col-sm-3 control-label">Total students:</label>
+            <div class="col-sm-6" id="total_students">
+                <p class="form-control-static">${courseDetails.stats.studentsTotal}</p>
+            </div>
+        </div>
+        
+        <div class="form-group">
+            <label class="col-sm-3 control-label">Instructors:</label>
+            <div class="col-sm-6" id="instructors">
+                <div class="form-control-static">
+                    <c:forEach items="${instructors}" var="instructor" varStatus="i">
+                        <c:choose>
+                            <c:when test="${empty instructor.role}">
+                                <%= Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER %>: ${instructor.name} (${instructor.email})
+                            </c:when>
+                            <c:otherwise>
+                                ${instructor.role}: ${instructor.name} (${instructor.email})
+                            </c:otherwise>
+                        </c:choose>
+                        <br>
+                        <br>
+                    </c:forEach>
+                </div>
+            </div>
+        </div>
+        
+        <c:if test="${courseDetails.stats.studentsTotal > 1}">
+            <div class="form-group">
```

Comment:
Would it be a good idea to extract this segment (the two buttons and the modal) further into another tag?
It seems this part of the page could be divided into three parts, the course information, the student buttons / list, and the comment functionality.